### PR TITLE
[WIP] added alternative database CouchDb - mergable

### PR DESCRIPTION
**Pull request by KharitonOff**
This PR continues the PR #518 ... 
For further development on this feature, please create a PR for the cla-assistant/Daimler-master branch.

### CODE REVIEW

File: src/server/api/cla.js [hunk 1]
```diff
@@ -524,9 +523,7 @@ async function claNotRequired(args, updateMethod) {
 // token (mandatory)
 async function updateUsersPullRequests(args) {
     try {
-        const user = await User.findOne({
-            name: args.user
-        })
+        const user = await userService.byUserName(args.name)
```
Comment (KharitonOff) [inline src/server/api/cla.js]: @codezork we used `args.user` and you changed it to `args. name` ... is it on purpose

File: src/server/api/cla.js [hunk 2]
```diff
@@ -582,7 +579,8 @@ async function prepareForValidation(item, user) {
     for (let i = needRemove.length - 1; i >= 0; --i) {
         user.requests.splice(needRemove[i], 1)
     }
-    user.save()
+    await userService.save(user);
```
Comment (KharitonOff) [inline src/server/api/cla.js]: @codezork do we need an `await` here? Can we just continue processing without waiting for user to be saved?

File: src/server/api/cla.js (hunk 1 above)
Comment (joerg-codecentric) [reply src/server/api/cla.js]: You are right. I messed it up. Corrected now

File: src/server/api/cla.js (hunk 2 above)
Comment (joerg-codecentric) [reply src/server/api/cla.js]: I found it more save. Meanwhile I found no reason to wait. Removed now.

**Comment by joerg-codecentric** [comment]
changes committed

**Pull request closed without merging**